modulegen/example_cmd.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewExampleCmd() *cobra.Command {
+	exampleVar := &ExampleVar{}
+	cmd := &cobra.Command{
+		Use:   "example",
+		Short: "Create a new Example",
+		Long:  "Create a new Example",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return generateExample(exampleVar, false)
```

Comment:
```suggestion
			return generateModule(exampleVar, true)
```

Reply:
If you are calling the method generateModule() it’s more clear to have asModule provided true than a asExample false . WDT ?

Reply:
I thought about it when posting, but seeing the following method signatures:

```go
func generateModule(e ExampleVar, asModule bool) error
```

and 

```go
func generateModule(e ExampleVar, asExample bool) error
``` 

I prefer the second. Is that what you mean?

Reply:
I propose a more simple way : 
```go
func generate(exampleVar ExampleVar, isModule bool) error
```